code/modules/cm_marines/orbital_cannon.dm

```diff
@@ -353,6 +353,9 @@ var/list/ob_type_fuel_requirements
 /obj/structure/ob_ammo/warhead
 	name = "theoretical orbital ammo"
 	var/warhead_kind
+	var/frequency
+	var/max_shake_factor
+	var/max_knockdown_time
```

Comment:
Could codedoc these (/// before each) though ultimately the minimum you should change is the name of `frequency` to something like `shake_frequency`